test/test_diffusion.py

```diff
@@ -319,43 +424,72 @@ def test_diffusion_accuracy(actx_factory, problem, nsteps, dt, scales, order,
 
         nodes = actx.thaw(dcoll.nodes())
 
+        if error_boundary:
+            target_boundary = error_boundary[0]
+            value = error_boundary[1]
+            alpha = error_boundary[2]
+        else:
+            value = None
+            alpha = None
+
         def get_rhs(t, u):
             kappa = p.get_kappa(nodes, t, u)
             if isinstance(kappa, DOFArray):
                 quadrature_tag = DISCR_TAG_QUAD
             else:
                 quadrature_tag = DISCR_TAG_BASE
+            boundaries = p.get_boundaries(dcoll, actx, t, value=value, alpha=alpha)
             return (
-                diffusion_operator(
-                    dcoll, kappa=kappa, boundaries=p.get_boundaries(dcoll, actx, t),
-                    u=u, quadrature_tag=quadrature_tag)
-                + evaluate(sym_f, x=nodes, t=t))
+                diffusion_operator(dcoll, kappa=kappa, boundaries=boundaries,
+                                   u=u, quadrature_tag=quadrature_tag)
```

Comment:
Triggering me here, lol :sweat: (not a big fan of the parentheses-aligned arguments style).